reporter-xray-udp/README.md

```diff
@@ -1,3 +1,69 @@
-# reporter-xray-udp
+# AWS X-Ray Reporter for Zipkin
 
-This sends json to XRay's daemon via UDP
\ No newline at end of file
+Used in an application that is being traced, this component converts Zipkin 
+trace data (spans) into [AWS X-Ray](https://aws.amazon.com/xray/)'s
+proprietary format and sends them to the 
+[X-Ray daemon](https://docs.aws.amazon.com/xray/latest/devguide/xray-daemon.html) 
+using UDP.
+
+Note that, unlike `AsyncReporter`, this reporter attempts to encode and send
+the span immediately on the calling thread.
+As UDP is used, there is no latency in waiting for the daemon to accept and 
```

Comment:
This is a bit misleading - there is plenty of latency in the I/O of just sending a UDP packet onto a socket using `java.net` APIs. If a server thread expects to not be blocked (e.g., it is a Netty event loop), then this reporter will need to be wrapped in a threadpool of some sort to make sure not to slow down the server if the write to network takes longer than usual as I/O syscalls may.

Ideally the implementation could be updated to use Netty or something, but for this documentation, I think it's better to avoid the term `no latency` and maybe say something like `As UDP is used, there is no time spent waiting for a response. However, this is still a blocking I/O operation so be careful when using this reporter in an asynchronous framework based on Netty.`

Reply:
Thanks for the improved wording. I adde it in, but left out the reference to Netty as it seems unrelated to this repo.